dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftRawLogsConnector.java

```diff
@@ -97,14 +99,21 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
         // 1. STL_QUERY has starttime, queryid, but text is 4000 char wich is useless
         // 2. STL_QUERY_TEXT has xid+squence+text which reconstructs query, but no starttime.
         // STL_QUERY is 1 row per query ; SQL_QUERY_TEXT is multi rows per query, using sequence and xid
-        String queryTemplateQuery
-                = "SELECT userid, xid, pid, query, trim(label) as label, starttime, endtime, sequence, text"
-                + " FROM STL_QUERY join STL_QUERYTEXT using (userid, xid, pid, query) WHERE ##";
+        List<String> queryTemplateColumns = new ArrayList<>(ImmutableList.<String>of(
```

Comment:
`Lists.newArrayList(E .... elements)` is somewhat neater

Reply:
Good point. I thought `newArrayList` was deprecated but that was just for the variant w/o arguments.